parser.y

```diff
@@ -1131,6 +1131,7 @@ import (
 	MaxIndexNumOpt                         "MAX_IDXNUM clause"
 	PerTable                               "Max index number PER_TABLE"
 	PerDB                                  "Max index number PER_DB"
+	EncryptionOpt                          "Encryption option 'Y' or 'N'"
```

Comment:
We can move `EncryptionOpt` declaration from `%type	<item>` to `%type	<ident>` to avoid unnecessary type conversion like `$4.(string)`.

``` 
%union {
	offset int // offset
	item interface{}
	ident string
	expr ast.ExprNode
	statement ast.StmtNode
}
```